employee_project/main.py

```diff
@@ -14,7 +25,7 @@ def main():
             name = input("Enter Employee Name: ")
             age = input("Enter Age:")
             ph_num = input("Enter Employee Phone Number: ")
-            address = input("Enter Employee Address : ")
+            address = emp_address()
```

Comment:
change method name emp_address() to get_emp_address()

Reply:
Changed